Reqnroll.VisualStudio/ProjectSystem/Settings/ReqnrollProjectSettingsProvider.cs

```diff
@@ -210,13 +210,27 @@ private NuGetPackageReference GetSpecFlowPackage(IProjectScope projectScope,
 
     private string GetReqnrollConfigFilePath(IProjectScope projectScope)
     {
-        var projectFolder = projectScope.ProjectFolder;
         var fileSystem = projectScope.IdeScope.FileSystem;
-        return fileSystem.GetFilePathIfExists(Path.Combine(projectFolder,
-                   ProjectScopeDeveroomConfigurationProvider.ReqnrollJsonConfigFileName)) ??
-               fileSystem.GetFilePathIfExists(Path.Combine(projectFolder,
-                   ProjectScopeDeveroomConfigurationProvider.SpecFlowJsonConfigFileName)) ??
-               fileSystem.GetFilePathIfExists(Path.Combine(projectFolder,
-                   ProjectScopeDeveroomConfigurationProvider.SpecFlowAppConfigFileName));
+        var assemblyDirectory = Path.GetDirectoryName(projectScope.OutputAssemblyPath);
+
+        var configFilePath = GetConfigFileInPath(fileSystem, projectScope.ProjectFolder);
+
+        if (assemblyDirectory != null)
+        {
+          configFilePath ??= GetConfigFileInPath(fileSystem, assemblyDirectory);
+          configFilePath ??= GetConfigFileInPath(fileSystem, Path.Combine(assemblyDirectory, "..", ".."));
```

Comment:
Why is this additional probing needed?

Reply:
I don't understand your question. I think it might be answered by the [comment above](https://github.com/reqnroll/Reqnroll.VisualStudio/pull/31/files#r1718920907).
If not, can you please rephrase it?

Reply:
Sorry. What I meant is that why isn't enough to check the config file in the assembly directory, why do we need to check the "<assemblyDirectory>\..\.." as well? So essentially why the line 

```
configFilePath ??= GetConfigFileInPath(fileSystem, Path.Combine(assemblyDirectory, "..", ".."));
```

is needed?

Reply:
This is done to match all the locations which which the runtime searches, see https://github.com/reqnroll/Reqnroll/blob/main/Reqnroll/Configuration/ReqnrollJsonLocator.cs

I don't specifically know why we would want to supporting having a config file there.

Reply:
I think in the runtime the original purpose of this was to get the project folder (this was the time when the usual output folder was just bin/debug).
Please remove that. It is surely obsolete.

Reply:
Sure, want me to also remove it from the runtime?

Reply:
Removed both here and in the runtime (see https://github.com/reqnroll/Reqnroll/pull/234)